[PROOFREADING] SID202 - Italian

AlbertoMagno21 opened the pull request:
I have reviewed the course SID202_it.

#1634

AlbertoMagno21 committed:
Update Introduction_SID202_it.md

Minimal change

AlbertoMagno21 committed:
Update chapter 1.1_it.md

Specified some English technical terms and correction small errors.

AlbertoMagno21 committed:
Update chapter 1.2_it.md

Explanation of English technical terms and small format corrections and changes to emphasize the importance of individual elements

AlbertoMagno21 committed:
Update it.md

AlbertoMagno21 committed:
Update chapter 2.1_it.md

Explanation of English technical terms and small format corrections and changes to emphasize the importance of individual elements

AlbertoMagno21 committed:
Update it.md

Little correction

AlbertoMagno21 committed:
Update chapter 2.2_it.md

Explanation of English technical terms and small format corrections and changes to emphasize the importance of individual elements

AlbertoMagno21 committed:
Update chapter 2.3_it.md

Explanation of English technical terms and small format corrections and changes to emphasize the importance of individual elements

AlbertoMagno21 committed:
Update chapter 3.1_ it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements and replaced "blind" with "offuscato".

AlbertoMagno21 committed:
Update chapter 3.2_it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements and replaced "blind" with blurring

AlbertoMagno21 committed:
Update chapter 3.3_it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements and replaced "blind" with "offuscato"

AlbertoMagno21 committed:
Update chapter 4.1_it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements and replaced "blind" with "offuscamento"

AlbertoMagno21 committed:
Update chapter 4.2_it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements and replaced "blind" with blurring

AlbertoMagno21 committed:
Update chapter 4.3_it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements and replaced "blind" with blurring

AlbertoMagno21 committed:
Update chapter 4.4_it.md

Explanation of English technical terms and small corrections and format changes to emphasize the importance of individual elements.

AlbertoMagno21 committed:
Update Course SID202_it.md

Formatting identical to the English version

AlbertoMagno21 committed:
Update it.md

AlbertoMagno21 committed:
Update it.md

Little change

Commit:
Correction of typos and technical words

Commit:
fix typos 2

MariJJhodl commented:
I applied some changes to the overall text, especially when it came to technical terms that had an unprecise translation. @AlbertoMagno21 can you re-read the text, this time only focusing on the flow of the text? To make sure it's easily readable in Italian

Asi0Flammeus committed:
Merge branch 'dev' into SID202_it

AlbertoMagno21 committed:
Update SID202_it.md

Line 67 eliminated a space before the point and eliminated after asset between redundant (asset) brackets.

AlbertoMagno21 committed:
Update SID202_it.md

Changed from "rese" to "resi".

AlbertoMagno21 committed:
Update SID202_it.md

Line 87 inverted "ampia" with "gamma"

AlbertoMagno21 committed:
Update Sid202_it.md

Line 103 I translated in brackets (Transazioni riservate)

AlbertoMagno21 committed:
Update SID202_it.md

Line 109 change "dei" to "degli"

AlbertoMagno21 committed:
Update SIS202_it.md

Line 113 deleted "s" from initialfreecoinss

AlbertoMagno21 committed:
Update SID202_it.md

Line 139 deleted "s" from Federations

AlbertoMagno21 committed:
Update SID202_it.md

line 152 translate watchmen to "guardiani" in brackets

AlbertoMagno21 committed:
Update SID202_it.md

Line 162 Change syntax

AlbertoMagno21 committed:
Update SID202_it.md

Line 162 change "basa" to "fonda"

AlbertoMagno21 committed:
Update SID202_it.md

Line 201 add "parametri" change "e" to "i".

Line 203 change "corso" with "svolgimento".

AlbertoMagno21 committed:
Update SID202_it.md

Line 304 translated "chiave di blocco" to blinding key

AlbertoMagno21 committed:
Update SID202_it.md

AlbertoMagno21 committed:
Update SID202_it.md

Line 404

AlbertoMagno21 committed:
Update SID202it.md

Line 414

AlbertoMagno21 committed:
Update SID202_it.md

Line 416

AlbertoMagno21 committed:
Update SID202it.md

Line 418

AlbertoMagno21 committed:
Update SID202_it.md

Line change 420 – 426 – 442 – 448 – 472 – 490 – 402 – 494 – 512 – 518 – 520

AlbertoMagno21 committed:
Update SID202_it.md

Asset in parentheses seemed redundant

AlbertoMagno21 committed:
Update SID202_it.md

Line 538

AlbertoMagno21 committed:
Update SID202_it.md

Line 540 - 542

AlbertoMagno21 committed:
Update SID202_it.md

Line 544

AlbertoMagno21 committed:
Update SID202_it.md

Chapther 3.3

AlbertoMagno21 committed:
Update SID202_it.md

AlbertoMagno21 committed:
Update SID202_it.md

Chapter 4.1

AlbertoMagno21 committed:
Update SID202_it.md

Chapter 4.2

AlbertoMagno21 committed:
Update SID202_it.md

Chapter 4.3

AlbertoMagno21 committed:
Update SID202_it.md

Chapter 4.4

AlbertoMagno21 reviewed (commented):
I ended up reviewing the PR by reporting all the technical names in English and trying to give more sense task to some phrases

AlbertoMagno21 reviewed (commented):
I ended up reviewing the PR by reporting all the technical names in English and trying to give more sense task to some phrases

Commit:
language fix

Commit:
Italian correction

Asi0Flammeus commented:
@MariJJhodl any update?

AlbertoMagno21 commented:
I've been a little busy these days, I've been carrying out the revision of the LNP201 course, but these days, already tonight I'll take it back.

AlbertoMagno21 committed:
Update Sid202_it.md

Little change

AlbertoMagno21 committed:
Update SID202_it.md

Little Change

AlbertoMagno21 committed:
Update SID202_it.md

Little Change

AlbertoMagno21 committed:
Update SID202_it.md

Little change - end

AlbertoMagno21 commented:
I ended up reviewing the PR by reporting all the technical names in English and trying to give more sense task to some phrases - course SID202_it.

#1634

MariJJhodl committed:
first check

MariJJhodl committed:
general check 2
